lib/membrane_h264_ffmpeg/parser.ex

```diff
@@ -99,23 +99,28 @@ defmodule Membrane.H264.FFmpeg.Parser do
                 description: """
                 Determines whether to drop the stream until the first set of SPS and PPS is received.
                 """
+              ],
+              max_b_frames: [
```

Comment:
The name of this option is a bit confusing - the parser does not enforce nor check it anyhow, it will just mess up the timestamps if the actual number of b frames exceeds this number.
Ideally, this option should not be needed at all as it's something that the parser should calculate. Considering this might not be as straightforward, I'd only change the name of this option, to better describe its meaning. In this case - I'd name it `max_reordered_frames` or `max_frame_reorder`